--- a/dlls/winemac.drv/cocoa_window.m
+++ b/dlls/winemac.drv/cocoa_window.m
@@ -982,7 +982,11 @@

     - (CALayer*) makeBackingLayer
     {
+#if defined(__x86_64__)
         CAMetalLayer *layer = [WineMetalLayer layer];   /* CW HACK 22435 */
+#else
+        CAMetalLayer *layer = [CAMetalLayer layer];
+#endif
         layer.device = _device;
         layer.framebufferOnly = YES;
         layer.magnificationFilter = kCAFilterNearest;
