--- a/loader/wine_info.plist.in
+++ b/loader/wine_info.plist.in
@@ -7,11 +7,11 @@
     <key>CFBundleDevelopmentRegion</key>
     <string>English</string>
     <key>CFBundleExecutable</key>
-    <string>wineloader</string><!-- CrossOver Hack 10913 -->
+    <string>secunda-wineloader</string>
     <key>CFBundleIdentifier</key>
-    <string>com.codeweavers.CrossOver.wineloader</string><!-- CrossOver Hack 10913 -->
+    <string>com.secundalauncher.wineloader</string>
     <key>CFBundleInfoDictionaryVersion</key>
     <string>6.0</string>
     <key>CFBundleName</key>
-    <string>CrossOver-Hosted Application</string><!-- CrossOver Hack 10913 -->
+    <string>Secunda Hosted Game</string>
     <key>CFBundlePackageType</key>
     <string>APPL</string>
     <key>CFBundleShortVersionString</key>
